tavern_pathfinder/src/character/mod.rs

```diff
@@ -0,0 +1,68 @@
+use serde::{Serialize,Deserialize};
```

Comment:
Future reference: you can make the same module by making the file `src/character.rs`, which is arguably better if there are not submodules.

Also don't forget to make sure `lib.rs` has a line saying `mod character;`.